ambry-router/src/main/java/com/github/ambry/router/PutOperation.java

```diff
@@ -161,7 +161,7 @@
 
   // Variables to keep track of netty chunks
   private final RestRequest restRequest;
-  private final String touchMessage;
+  private final String loggingContext;
```

Comment:
Looks like you want to print channel information. How about name it to channelContext ?

Reply:
I also include some other information like encryption. And this is per-putoperation context for logging. I think loggingContext is fine.